### ISSUE TITLE
is there any case when a clob is converted to bytea instead of text?

### ISSUE THREAD

**Issue by 3kkani** [darold/ora2pg#235]
Hello Gilles,

I am attempting the migration of an atlassian confluence schema from Oracle to PostgreSQL with ora2pg. 

The source DB is us7ascii but contains also characters in other encodings (it seems mainly iso-8859 but we really do not know).

The direct loading into pg with automatic character set conversion is failing to convert some characters and generating exceptions.

So, we are copying the data into files, converting them to utf-8 (using iconv and some ad hoc python scripts) and loading them into pg. 

I can see that blobs are mapped to bytea and clobto text but one clob column is puzzling me because it is converted to bytea for unknown reasons.

Can you help me understanding if that is an expected behavior, a feature, a bug, something depending on the content/data or may be just the result of a misconfiguration?

Many thanks.
Maurizio.

**Comment by darold** [comment]
Hi Maurizio,

Please post here the result of the following command:

    ora2pg -c .../ora2pg.conf -t SHOW_COLUMN -a 'your_table_name'

This will show us the types detected by Ora2Pg and the conversion type.

**Comment by 3kkani** [comment]
Hi Gilles, 

below is the result of the command but do not worry about that. The fault was mine and due to a misconfiguration. Something that I tried some days ago to solve a character set conversion issue.

The column in question is a clob. The content looks like a json collection of some i18n multi-language messages. 

Converting the related file with iconv from iso-8859-1 to UTF-8 is apparently solving the conversion errors (no more exceptions) but some characters (JP for example) does not look like they are converted/visualized correctly at the end.

So I tried to avoid any conversion at all by forcing the mapping of the column as bytea even though the application might not like it. I am now investigating if the table can actually be re-populated completely from scratch by some configuration/settings option of the application software.

Thanks for your help and for ora2pg that is a quite powerful, versatile and complete tool.

[1] TABLE AO_9412A1_AOREGISTRATION (owner: CONFLUENCE, 3 rows)
	DATA : CLOB(4000) => bytea
	ID : VARCHAR2(255) => varchar(255)
	UPDATED : TIMESTAMP(6)(11) => timestamp
----------------------------------------------------------
Total number of rows: 3

Top 10 of tables sorted by number of rows:
	[1] TABLE AO_9412A1_AOREGISTRATION has 3 rows
Top 10 of largest tables:
	[1] TABLE AO_9412A1_AOREGISTRATION: 0 MB (3 rows)
